attendance/resources/attendance.py

```diff
@@ -10,9 +12,17 @@ def get(self):
 
     def post(self):
         body = request.get_json()
-        attendance = Attendance(**body).save()
-        id = attendance.id
-        return {'id': str(id)}, 200
+        attendance = Attendance(**body)
+        apiResponse = urllib.request.urlopen("http://management/manage/geo-fence?universityName="+attendance.school).read()
+        responseData = json.loads(apiResponse.decode('utf8')).get("data")
+        if responseData is None:
+            return {'error': "School does not exist in db"}, 500
+        distance = geodesic(body.get("location"),[responseData.get("latitude"), responseData.get("longitude")]).m
+        if distance < responseData.get("radiusInMeter") :
+            attendance.save()
+        else:
+            return {'error': "Not in the premise area"}, 500
+        return {'id': str(attendance.id)}, 200
```

Comment:
Need to build capability to send reportable information to Reporting service through Kafka / Direct API calls.